setup.sh

```diff
@@ -24,63 +32,86 @@ if [ ! -e ./.env ]; then
 	value_default="kopano.demo"
 	read -p "FQDN to be used (for reverse proxy) [$value_default]: " new_value
 	FQDN=${new_value:-$value_default}
+	LDAP_BASE_DN=$(fqdn_to_dn $FQDN)
 
 	value_default="self_signed"
 	read -p "Email address to use for Lets Encrypt. Use 'self_signed' as your email to create self signed certificates [$value_default]: " new_value
 	EMAIL=${new_value:-$value_default}
 
-	value_default="dc=kopano,dc=demo"
+	value_default="$LDAP_BASE_DN"
 	read -p "Name of the BASE DN for LDAP [$value_default]: " new_value
 	LDAP_BASE_DN=${new_value:-$value_default}
 
-	value_default="kopano123"
-	read -p "Password of the admin user (in bundled LDAP) [$value_default]: " new_value
-	LDAP_ADMIN_PASSWORD=${new_value:-$value_default}
-
 	value_default="ldap://ldap:389"
 	read -p "LDAP server to be used (default to bundled openldap) [$value_default]: " new_value
 	LDAP_SERVER=${new_value:-$value_default}
 
-	value_default="DC=kopano,DC=demo"
-	read -p "LDAP search base [$value_default]: " new_value
-	LDAP_SEARCH_BASE=${new_value:-$value_default}
-
-	value_default="CN=readonly,DC=kopano,DC=demo"
-	read -p "LDAP bind user (needs only read permissions) [$value_default]: " new_value
-	LDAP_BIND_DN=${new_value:-$value_default}
-
-	value_default="kopano123"
-	read -p "LDAP server to be used (default bundled openldap) [$value_default]: " new_value
-	LDAP_BIND_PW=${new_value:-$value_default}
+	if [ "$LDAP_SERVER" != "$value_default" ]; then
+		value_default="kopano123"
+		read -p "Password of the admin user [$value_default]: " new_value
```

Comment:
When not using the bundled ldap then we do not have to differentiate between  admin/read-only user. remove the ldap_admin_password option.